From b2ac1530fd8cfc2f1da5c9817f2babc5a7bd2068 Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Mon, 1 Jun 2015 17:18:27 +0800
Subject: [PATCH] win32/configure.js: Fix Build

This ought to look for configure.ac instead of the old-style configure.in,
as we switched to it for some time.
---
 win32/configure.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/win32/configure.js b/win32/configure.js
index edd943a..92b9ba0 100644
--- a/win32/configure.js
+++ b/win32/configure.js
@@ -14,7 +14,7 @@ var srcDirUtils = "..";
 var baseName = "libxml2";
 /* Configure file which contains the version and the output file where
    we can store our build configuration. */
-var configFile = srcDirXml + "\\configure.in";
+var configFile = srcDirXml + "\\configure.ac";
 var versionFile = ".\\config.msvc";
 /* Input and output files regarding the libxml features. */
 var optsFileIn = srcDirXml + "\\include\\libxml\\xmlversion.h.in";
-- 
2.3.5.windows.8

